linux/buildScript.sh

```diff
@@ -11,12 +11,12 @@ set -u
 
 srctgz="https://www.census.gov/ts/x13as/unix/x13ashtmlsrc_V1.1_B19.tar.gz"
 file=$(basename ${srctgz})
-
 td=$(tempfile -p x13)
+rm -r $td
 mkdir ${td}
 cd ${td}
 wget ${srctgz}
-tar xfz ${file}
+tar xf ${file}
```

Comment:
I am a big fan of using eg `tar xaf ...` to infer the compression but was under impression that the extension overruled the magic number.  But not so as you point out:

```
edd@max:/tmp/Rtmp0vmNby$ tar taf x13ashtmlsrc_V1.1_B26.tar.gz | head
aaamain.f
abend.f
acfar.f
acfdgn.f
acf.f
Acf.f
acfhdr.f
addadj.f
addaic.f
addate.f
edd@max:/tmp/Rtmp0vmNby$ 
```

So `a` instead of `z` it is. Good.  (And yes, still a braindead file...)